repo_checker.py

```diff
@@ -39,14 +39,16 @@ def __init__(self, *args, **kwargs):
 
         # RepoChecker options.
         self.skip_cycle = False
+        self.force = False
+        self.limit_group = None
 
     def project_only(self, project, post_comments=False):
         # self.staging_config needed by target_archs().
         api = self.staging_api(project)
 
         root = ET.fromstringlist(show_results_meta(
             self.apiurl, project, multibuild=True, repository=['standard']))
-        if len(root.xpath('result[@state!="published"]')):
+        if not self.force and len(root.xpath('result[@state!="published"]')):
```

Comment:
I have suggest perhaps a "--skip-checks" or somesuch to skip all the checks in that section as sometimes helpful when manually running, but "--force" seems a bit ambiguous and ends up only forcing one check.